### PR TITLE
ref(crumbs): Improve crumbs doc

### PR DESCRIPTION
**Pull request by priscilawebdev**
closes https://app.asana.com/0/1182975495223914/1190535597492307

This PR improves the Breadcrumbs interface document. 

I left to add the new `system` and `session` types when the mobile team makes it available.

### CODE REVIEW

File: src/docs/sdk/event-payloads/breadcrumbs.mdx [hunk 1]
```diff
@@ -2,107 +2,149 @@
 title: Breadcrumbs Interface
 ---
 
-The Breadcrumbs Interface specifies a series of application events, or
-"breadcrumbs", that occurred before an event.
+Sentry uses _breadcrumbs_ to create a trail of events that happened prior to an issue. These events are very similar to traditional logs but can record more rich structured data.
```
Comment (HazAT) [inline src/docs/sdk/event-payloads/breadcrumbs.mdx:5]:
```suggestion
Sentry uses _breadcrumbs_ to create a trail of events that happened before an issue. These events are very similar to traditional logs but can record more rich structured data.
```

File: src/docs/sdk/event-payloads/breadcrumbs.mdx [hunk 2]
```diff
@@ -2,107 +2,149 @@
...
+  "event_id": null,
+  "level": "info",
+  "message": "this is a message",
+  "timestamp": "2020-08-28T07:29:37Z"
+}
+```
+
+### `exception`
+
+The occurrence of an expection.
```
Comment (HazAT) [inline src/docs/sdk/event-payloads/breadcrumbs.mdx:86]:
```suggestion
The occurrence of an exception.
```

File: src/docs/sdk/event-payloads/breadcrumbs.mdx [hunk 3]
```diff
@@ -2,107 +2,149 @@
...
+
+```json
+{
+  "entries": [
+    { "type": "exception", "data": {...} },
+    { "type": "breadcrumbs", "data": {...} }]
+}
+```
+
+The `data` object of an expection will always contain the `mechanism` property describing if the expection was handled or not.
```
Comment (HazAT) [inline src/docs/sdk/event-payloads/breadcrumbs.mdx:98]:
```suggestion
The `data` object of an exception will always contain the `mechanism` property describing if the exception was handled or not.
```

File: src/docs/sdk/event-payloads/breadcrumbs.mdx [hunk 4]
```diff
@@ -2,107 +2,149 @@
...
+  "stacktrace": null,
+  "threadId": null,
+  "type": "TypeError",
+  "value": "Failed to fetch"
+}
+```
+
+### `error`
+
+An error that occurred prior to the exception.
```
Comment (HazAT) [inline src/docs/sdk/event-payloads/breadcrumbs.mdx:114]:
```suggestion
An error that occurred before the exception.
```

File: src/docs/sdk/event-payloads/breadcrumbs.mdx [hunk 5]
```diff
@@ -112,45 +154,129 @@ Its `data` property has the following sub-properties:
...
     "reason": "OK"
-  }
+  },
+  "timestamp": "2016-04-20T20:55:53.845Z"
+}
+```
+
+### `info`
+
+An information that could help you identify the root cause of the issue or who the error happened for.
```
Comment (HazAT) [inline src/docs/sdk/event-payloads/breadcrumbs.mdx:192]:
```suggestion
Information that could help you identify the root cause of the issue or who the error happened for.
```

**Review by HazAT — CHANGES_REQUESTED** [review]

File: src/docs/sdk/event-payloads/breadcrumbs.mdx (hunk 1 above)
Comment (priscilawebdev) [reply src/docs/sdk/event-payloads/breadcrumbs.mdx:5]: I like your suggestion but I will leave it as it is to be consistent with the breadcrumbs description that we have on docs.sentry https://docs.sentry.io/enriching-error-data/breadcrumbs/

**Review by untitaker — APPROVED** [review]

**Review by HazAT — APPROVED** [review]

File: src/docs/sdk/event-payloads/breadcrumbs.mdx (hunk 1 above)
Comment (PeloWriter) [reply src/docs/sdk/event-payloads/breadcrumbs.mdx:5]: +1 on adopting this intro. We should move to it for all breadcrumbs docs.

File: src/docs/sdk/event-payloads/breadcrumbs.mdx [hunk 6]
```diff
@@ -2,107 +2,146 @@
...
 
-: _Optional_. This defines the severity level of the breadcrumb. Allowed values
-  are, from highest to lowest: `fatal`, `error`, `warning`, `info`, and `debug`.
-  Levels are used in the UI to emphasize and deemphasize the crumb. Defaults to
-  `info`.
+Contains a dictionary whose contents depend on the breadcrumb type. Additional parameters that are unsupported by the type are rendered as a key/value table.
+
+`level` (optional)
+
+This defines the severity level of the breadcrumb. Allowed values are, from highest to lowest: `fatal`, `error`, `warning`, `info`, and `debug`. Levels are used in the UI to emphasize and deemphasize the crumb. Defaults to `info`.
```
Comment (PeloWriter) [inline src/docs/sdk/event-payloads/breadcrumbs.mdx:67]:
```suggestion
: This defines the severity level of the breadcrumb. Allowed values are, from highest to lowest: `fatal`, `error`, `warning`, `info`, and `debug`. Levels are used in the UI to emphasize and deemphasize the crumb. The default is `info`.
```

File: src/docs/sdk/event-payloads/breadcrumbs.mdx [hunk 7]
```diff
@@ -112,68 +151,110 @@ Its `data` property has the following sub-properties:
...
+  "timestamp": "2016-04-20T20:55:53.845Z"
 }
 ```
 
-## Examples
+### `info`
 
-The following example illustrates the breadcrumbs part of the <Link to="/sdk/event-payloads/">event
-payload</Link> and omits other attributes for simplicity.
+Information that could help you identify the root cause of the issue or who the error happened for.
```
Comment (PeloWriter) [inline src/docs/sdk/event-payloads/breadcrumbs.mdx:190]:
```suggestion
Information that helps identify the root cause of the issue or for whom the error occurred.
```

File: src/docs/sdk/event-payloads/breadcrumbs.mdx [hunk 8]
```diff
@@ -2,107 +2,146 @@
...
+`type` (optional)
 
-: _Recommended_. A timestamp representing when the breadcrumb occurred. The
-  format is either a string as defined in [RFC
-  3339](https://tools.ietf.org/html/rfc3339) or a numeric (integer or float)
-  value representing the number of seconds that have elapsed since the [Unix
-  epoch](https://en.wikipedia.org/wiki/Unix_time).  
-  Breadcrumbs are most useful when they include a timestamp, as it creates a
-  timeline leading up to an event.
+The type of breadcrumb.
```
Comment (PeloWriter) [inline src/docs/sdk/event-payloads/breadcrumbs.mdx:43]:
```suggestion
: The type of breadcrumb. By default, all breadcrumbs are recorded as `default`, which makes them appear as a Debug entry, but Sentry provides other types that influence how the breadcrumbs are rendered. For more information, see the [description of recognized breadcrumb types](#breadcrumb-types).
```

File: src/docs/sdk/event-payloads/breadcrumbs.mdx [hunk 9]
```diff
@@ -2,107 +2,146 @@
...
 
-`type`
+By default, all breadcrumbs are recorded as `default`, which makes them appear as a Debug entry, but Sentry provides other types that influence how the breadcrumbs are rendered. For more information, see the [description of recognized breadcrumb types](#breadcrumb-types).
```
Comment (PeloWriter) [inline src/docs/sdk/event-payloads/breadcrumbs.mdx:45]: added to above

File: src/docs/sdk/event-payloads/breadcrumbs.mdx (hunk 8 above)
Comment (PeloWriter) [reply src/docs/sdk/event-payloads/breadcrumbs.mdx:43]: Note the tiny change here that renders the term and definition for easier reading. A good example of this is in src/docs/platform/javascript/common/config/basic.mdx

Tl;dr Use this format:
`term`

: definition

The term is then bold and in code format with the definition that follows slightly indented.

File: src/docs/sdk/event-payloads/breadcrumbs.mdx [hunk 10]
```diff
@@ -2,107 +2,146 @@
...
 
-: _Optional_. The type of breadcrumb. The default type is `default`, which
-  indicates no specific handling. Other types are currently `http` for HTTP
-  requests and `navigation` for navigation events. For more information, see the
-  [description of recognized breadcrumb types](#breadcrumb-types).
+`category` (optional)
 
-`category`
+A dotted string indicating what the crumb is or from where it comes. Typically it is a module name or a descriptive string. For instance, `ui.click` could be used to indicate that a click happened in the UI or flask could be used to indicate that the event originated in the Flask framework.
```
Comment (PeloWriter) [inline src/docs/sdk/event-payloads/breadcrumbs.mdx:49]:
```suggestion
: A dotted string indicating what the crumb is or from where it comes. Typically it is a module name or a descriptive string. For instance, `ui.click` could be used to indicate that a click happened in the UI or flask could be used to indicate that the event originated in the Flask framework.
```

File: src/docs/sdk/event-payloads/breadcrumbs.mdx [hunk 11]
```diff
@@ -2,107 +2,146 @@
...
-  _flask_ could be used to indicate that the event originated in the Flask
-  framework.
+Internally we render some crumb’s color and icon based on the provided category. For more information, see the [description of recognized breadcrumb types](#breadcrumb-types).
 
-`message`
+`message` (optional)
 
-: _Optional_. If a message is provided, it is rendered as text with all
-  whitespace preserved. Very long text might be truncated in the UI.
+Human readable message for the breadcrumb.
```
Comment (PeloWriter) [inline src/docs/sdk/event-payloads/breadcrumbs.mdx:55]:
```suggestion
: Human readable message for the breadcrumb.
```

File: src/docs/sdk/event-payloads/breadcrumbs.mdx [hunk 12]
```diff
@@ -2,107 +2,146 @@
...
+If a message is provided, it is rendered as text with all whitespace preserved.
 
-: _Optional_. Arbitrary data associated with this breadcrumb. Contains a
-  dictionary whose contents depend on the breadcrumb `type`. See the
-  [description of breadcrumb types](#breadcrumb-types). Additional parameters
-  that are unsupported by the type are rendered as a key/value table.
+`data` (optional)
 
-`level`
+Arbitrary data associated with this breadcrumb.
```
Comment (PeloWriter) [inline src/docs/sdk/event-payloads/breadcrumbs.mdx:61]:
```suggestion
: Arbitrary data associated with this breadcrumb.
```

File: src/docs/sdk/event-payloads/breadcrumbs.mdx [hunk 13]
```diff
@@ -2,107 +2,146 @@
...
+A timestamp representing when the breadcrumb occurred. The format is either a string as defined in [RFC 3339](https://tools.ietf.org/html/rfc3339) or a numeric (integer or float) value representing the number of seconds that have elapsed since the [Unixepoch](https://en.wikipedia.org/wiki/Unix_time).  
+Breadcrumbs are most useful when they include a timestamp, as it creates a timeline leading up to an event `expection/error`.
 
 ## Breadcrumb Types
 
-Below are descriptions of individual breadcrumb types, and what their `data`
-properties look like.
+Below are descriptions of individual breadcrumb types, and what their `data` properties look like.
 
 ### `default`
```
Comment (PeloWriter) [inline src/docs/sdk/event-payloads/breadcrumbs.mdx:78]: rather than setting these as H3, I'd recommend using 
`term`
: description

as the format for this and the following terms for easier readability. You'll need to move the margin in for the code samples so they align as well.

Comment (priscilawebdev) [reply src/docs/sdk/event-payloads/breadcrumbs.mdx:78]: I tried to align the samples and phrases that start with "Internally ..." by adding some indentation, but that doesn't seem possible as it still displays the same or the json as a text. Please see below:

image

Comment (MimiDumpling) [reply src/docs/sdk/event-payloads/breadcrumbs.mdx:78]: Hi @priscilawebdev, thanks for helping with docs. I'm taking a look at this and may have a solution shortly. 😸
